src/api/users.js

```diff
@@ -0,0 +1,6 @@
+import { client } from "@/utils/fetchClient";
+
+export const getexistingUser = (userEmail) =>
```

Comment:
Consider renaming `getexistingUser` to `getExistingUser` to follow camelCase naming conventions consistently.